mhs/outbound/main.py

```diff
@@ -140,7 +140,7 @@ def main():
                                                   ca_certs=secrets.get_secret_config('CA_CERTS'))
     max_retries = int(config.get_config('OUTBOUND_TRANSMISSION_MAX_RETRIES', default="3"))
     retry_delay = int(config.get_config('OUTBOUND_TRANSMISSION_RETRY_DELAY', default="100"))
-    validate_cert = bool(config.get_config('OUTBOUND_VALIDATE_CERTIFICATE', default="True"))
+    validate_cert = config.get_config('OUTBOUND_VALIDATE_CERTIFICATE', default=str(True)).lower() == str(True).lower()
```

Comment:
`default='true').lower() == 'true'` would be simpler. Is there a reason its written this way?

Reply:
just to omit potential error in typing a string
anyway, I've added a str2bool function